### ISSUE TITLE
cannot create new member when dataset type is PDSE

### ISSUE THREAD

**Issue by nk2164** [zowe/vscode-extension-for-zowe#407]
I tried creating a new member in a PDSE library dataset i created on the mainframe. The member creation fails with a time out . If the dataset is of type PDS , the member creation works fine.

`Error running command zowe.createMember: z/OSMF REST API Error:
Rest API failure with HTTP(S) status 500
rc:       4
reason:   1
category: 2
message:  receiveResponseHeader: timeout receiving response (>30 secs): TsoServerConnection(USER=Zxxxx, ASID=0x0065, QID=0x000c000b)

. This is likely caused by the extension that contributes zowe.createMember.

Unable to create member: z/OSMF REST API Error:
Rest API failure with HTTP(S) status 500
rc:       4
reason:   1
category: 2
message:  receiveResponseHeader: timeout receiving response (>30 secs): TsoServerConnection(USER=Zxxxx, ASID=0x0065, QID=0x000c000b)`

**Comment by jellypuno** [comment]
I can create a member in my PDSE library

**Comment by zFernand0** [comment]
@jellypuno, is there a Milestone we can put this one in? 
Just for awareness for people running into this problem and maybe just need to upgrade their version to that of the milestone?

**Comment by jellypuno** [comment]
@zFernand0 I'll put it 1.3.0. That's the version that I have right now and its working fine.

**Comment by nk2164** [comment]
@jellypuno @zFernand0 Thanks. I'll try with that version.

**Comment by zFernand0** [comment]
Hey @nk2164, 
Any luck?

**Comment by nk2164** [comment]
@zFernand0 Not yet. Is it possible to create a new PDSE dataset from the extension ? Last time i tried , i had an already allocated a PDSE dataset on the zowe tutorial system. I had created that by logging in via TN3270 emulator from the zlux desktop. I lost that one (need to rebuild it sometime this week). I tried creating a new PDSE dataset from the plugin and got errors. I will get back to you sometime next week.
